assets/js/src/core/components/grid/columns/types/textarea/textarea-cell.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { type CellContext } from '@tanstack/react-table'
+import React, { useState } from 'react'
+import { useEditMode } from '../../../edit-mode/use-edit-mode'
+import { useStyle } from './textarea-cell.styles'
+import TextArea from 'antd/es/input/TextArea'
+import { respectLineBreak } from '@Pimcore/utils/helpers'
+
+export type DefaultCellProps = CellContext<any, any>
```

Comment:
I would prefer to use the type from the DefaultCell: https://github.com/pimcore/studio-ui-bundle/blob/1.x/assets/js/src/core/components/grid/columns/default-cell.tsx#L27

Otherwise we miss some type completion. :)

Reply:
Copied this from the text-cell -> I changed it now in both components